### ISSUE TITLE
Remember witch episode you last watched on the "watch S E" button

### ISSUE THREAD

**Issue by oathzed** [stantanasi/streamflix#72]
### Description

If I haven't watched a season and I just want to start from s2 e5 for example. If I start that episode and then press back. It will say my episode to watch is s1e1, when it should say I've watched 5 minutes of s2e5 and I can quickly watch it from that button. This is how Netflix does it, so you can pick up from where you left off. This is useful for shows like friends or the office where most people rewatch random episodes.

Your next episode should also be based off what episode you previously watched

Check video for what happens:

https://imgur.com/a/5myEQ06

### Reproduction steps

1. Go to "the rookie" season 2 episode 3, play it to 15min
2. Press back, see that you've watched part of it, press back
3. Home screen will say watch s1e1

### App version

Latest

### Operating system

Shield tv

### Please confirm the following

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] If related to a provider, I have checked the site and it works, but not the app.

**Comment by stantanasi** [comment]
Alright, let me clarify:
For instance, if you've watched 15 episodes of a show, the app will display "Watch S1 E16" to watch the next episode. If you decide to rewatch episode 3, you'd expect to see "Watch S1 E3" to resume that specific episode. Once you've finished watching it, the app will revert back to displaying "Watch S1 E16".

Is it how you want it?

**Comment by oathzed** [comment]
> Alright, let me clarify: For instance, if you've watched 15 episodes of a show, the app will display "Watch S1 E16" to watch the next episode. If you decide to rewatch episode 3, you'd expect to see "Watch S1 E3" to resume that specific episode. Once you've finished watching it, the app will revert back to displaying "Watch S1 E16".
> 
> Is it how you want it?

Not quite. Let's say you haven't watched any of it, but you decide to start on season 3 episode 1. If you watch part of that episode, have it remember that and on the tv show page have it display "Resume S3E1".

Now after I finished watching S3E1, it will display "Watch S3E2".

If I decide I want to start the show back from the beginning (S1E1), if I watch that episode half way, and go back to that tv show page, it will say "Resume S1E1.

Now if I watch S1E1 all the way to the end, it will display "Watch S1E2" on the tv page.

TLDR: in short no matter where you watch the show from it will display what episode you were last watching (having that resume point), and what ever episode you were last watching that you finished, it will display the episode after that.

Side note. If you have watch every episode to the end. And the next week a new episode comes out, have that episode pop up in continue watching and also have it in the show page as "watch (latest episode number)"

**Comment by stantanasi** [comment]
**Available on [Release 1.5.1](https://github.com/stantanasi/streamflix/releases/latest)**
